<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1010</width>
      <height>730</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Публикации</string>
     </attribute>
     <widget class="QLineEdit" name="gamePublicationsText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Игра</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="publisherPublicationsText">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Издатель</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="creatorPublicationText">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>30</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Разработчик</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="publicationsTable">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>911</width>
        <height>530</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="datePublicationsText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>630</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Дата публикации</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="platformPublicationsText">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>630</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Платформа</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPublicationsButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletePublicationsButton">
      <property name="geometry">
       <rect>
        <x>801</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Игры</string>
     </attribute>
     <widget class="QLineEdit" name="nameGamesText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="genreGamesText">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Жанр</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="versionGamesText">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Версия</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addGamesButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteGamesButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="gamesTable">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>930</width>
        <height>530</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="genreSearchGamesText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>630</y>
        <width>570</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Поиск по жанру</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchGamesButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteSearchGamesButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Издатели</string>
     </attribute>
     <widget class="QLineEdit" name="namePublishersText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="countryPublishersText">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Страна</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="datePublishersText">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Дата основания</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPublishersButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletePublishersButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="publishersTable">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>930</width>
        <height>530</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="countrySearchPublishersText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>630</y>
        <width>570</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Поиск по стране</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchPublishersButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteSearchPublishersButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Разработчики</string>
     </attribute>
     <widget class="QLineEdit" name="nameCreatorsText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="countryCreatorsText">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Страна</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailCreatorsText">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Почта поддержки</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addCreatorsButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteCreatorsButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>30</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="creatorsTable">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>930</width>
        <height>530</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="countrySearchCreatorsText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>630</y>
        <width>570</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Поиск по стране</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchCreatorsButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteSearchCreatorsButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>630</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>База данных</string>
    </property>
    <addaction name="connectAction"/>
    <addaction name="dropAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Таблицы</string>
    </property>
    <addaction name="clearAction"/>
    <addaction name="clearAllAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="connectAction">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="dropAction">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>Очистить текущую</string>
   </property>
  </action>
  <action name="clearAllAction">
   <property name="text">
    <string>Очистить все</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
